src/pages/Checkmark.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './Checkmark.css';
+
+function Checkmark() {
+  return <input type="checkbox"></input>;
+}
+
+export default Checkmark;
```

Comment:
Wait, it looks like you didn't end up using this component. Is that correct? If so, please delete it. Thanks!